source/technical/reservations.rst

```diff
@@ -128,6 +133,44 @@ You may reserve a specific node by providing its *UUID*. To learn more about how
 
 .. _reservations-extend-lease-gui:
 
+Creating a Lease to Reserve a VLAN Segment
+__________________________________________
+
+On the *Create Lease* window, start filling the form as if reserving physical
+nodes: pick a name, start time, and lease duration. Then:
+
+#. Change the *Resource Type* field to *Network*.  The control to configure the
+   number of hosts will disappear and be replaced by a field to enter the
+   network name.
+
+#. Enter a name for the Neutron network which will be created by Blazar on the
+   reserved VLAN segment.
+
+#. Remove any resource properties.
+
+   .. note:: In the future the controls for *Resource Properties* will be improved to show only those relevant to VLANs.
+
+#. Click on the *Create* button.
+
+Once created, the lease details will be displayed. At the bottom of the page
+are the details about the reservation. Initially the reservation is in the
+``Pending`` status, and stays in this state until it reaches the start time.
+
+.. figure:: reservations/leasedetails_vlan.png
+   :alt: Lease details page for a VLAN reservation
+
+   Lease details page for a VLAN reservation
+
+Once the start time of the lease is reached, the lease will be started and its
+reservation will change to ``Active``; you may need to refresh the page to see
+the updates.
+
+At this stage, Blazar will create a new Neutron network using the reserved VLAN
+segment and the name entered in the form. You can then use Horizon to create a
+subnet, attach a router, and launch instances using the Neutron network created
+by Blazar.
+
+.. note:: When a VLAN segment reservation ends, all Neutron resources attached to the network will be automatically deleted. Bare-metal instances using the network will lose network connectivity.
```

Comment:
Does this mean that the instance will be running (active), but the network resources will be deleted? If so, is it possible to delete a network when an active instance is attached to it?

Reply:
The instance will keep running but will be disconnected from the network. `networking-generic-switch` will reconfigure the bare-metal port to move it to the default VLAN (generally 1).